Submissions/kademgari_week2.Playground 1 Simple Values and Control Flow.playground/Contents.swift

```diff
@@ -0,0 +1,75 @@
+//: Simple Values
+//: 1. Create a constant with an explicit type of Float that equals 4.
+let explicitfloat: Float = 4
+//: 2. Try removing the conversion to String from the last line. Copy the error and place it in a block comment below this block of code.
+let label = "The width is "
+let width = 94
+let widthLabel = label + String(width) //  With out String following error is generated
+
+/* error: kademgari_Assign2.playground:5:24: error: binary operator '+' cannot be applied to operands of type 'String' and 'Int'
+ let widthlabel = label + width
+                  ~~~~~ ^ ~~~~~     */
+//: 3. Use \() to include a floating-point calculation in a string and to include someone's name in a greeting.
+let radiofrequency: Float = 101
+let name = "steve"
+let message = "Hi can you please tune the radio to  \(radiofrequency+0.5). Thanks \(name)!"  //"Hi can you please tune the radio to 101.5. Thanks steve!"
+//: Control Flow
+//: 1. Change optional Name to nil. What greeting do you get? Add an else clause that sets a different greeting if optionalName is nil.
+var optionalString: String? = "Hello"
+print(optionalString == nil)     //prints false
+
+var optionalName: String? = nil
+var greeting = "Hello!"
+if let name = optionalName {
+    greeting = "Hello, \(name)"}   //This prints "Hello,"
```

Comment:
If `optionalName` is nil, this block will not execute and will not print "Hello,".